### DISCUSSION TITLE
大佬们，还有其他客户端吗

### DISCUSSION THREAD

**Discussion by iwalke** [MetaCubeX/ClashMetaForAndroid#284] [General]
大佬们，Android 还有其他客户端吗

[FlClash](https://github.com/chen08209/FlClash) 这个怎么样，和 ClashMetaForAndroid 相比哪个好用一些

**Comment by XiandaLyu** [comment]
> 大佬们，Android 还有其他客户端吗
> 
> [FlClash](https://github.com/chen08209/FlClash) 这个怎么样，和 ClashMetaForAndroid 相比哪个好用一些
> 

FL不能更新play store

**Reply by fuwuqi298** [reply]
部好用，从那个转过来的。同一个机场，那个会断流，有时候打开app，启动代理，状态栏 vpn 的标都出不来。

**Reply by fuwuqi298** [reply]
ClashMetaForAndroid 有个问题，点近期右边那三个点，好像有弹窗出不来。
![a125fcd2a527259d80a172c6719a9c2](https://github.com/user-attachments/assets/8f32f83e-fc3f-47b0-9dbf-07aebc162966)

**Reply by 743859910** [reply]
正常的![Screenshot_2024-09-16-04-36-03-285_com.github.metacubex.clash.meta.jpg](https://github.com/user-attachments/assets/7c1d6533-cff3-4269-b8f5-6f34722fd9cc)

**Reply by HozumiMoa** [reply]
> > 大佬们，Android 还有其他客户端吗
> > [FlClash](https://github.com/chen08209/FlClash) 这个怎么样，和 ClashMetaForAndroid 相比哪个好用一些
> 
> FL不能更新play store

不能更新 play store 是规则的问题，跟 app 无关

**Reply by sudot** [reply]
![image](https://github.com/user-attachments/assets/76acd3a0-9a58-479d-b471-4862efb8e41f)

![image](https://github.com/user-attachments/assets/859fa069-0e89-4af5-8ed9-c64d56b3d1b7)

FlClash内存占用太大了，不开主界面占用200多兆，打开主界面之后点几下就600多兆内存
